src/racing/math_util.h

```diff
@@ -60,7 +60,7 @@ u16  random_u16(void);
 u16  random_int(u16);
 s16  func_802B7F34(f32, f32, f32, f32);
 void func_802B7F7C(Vec3f, Vec3f, Vec3s);
-f32  sins(u16);
+f32  sins(const u16);
```

Comment:
what is the advantage to use const for sins and not for coss ?

Reply:
I believe the arg for coss gets modified and sins does not. It's how I received the code. And just going off my memory because I think I had the same question.